### PR TITLE
Translation pattern form

### PR DESCRIPTION
**Pull request by shenlin192**
Hello @choudlet @afc163, my translation for the form in pattern is finished. Please have a look if you are free.

### CODE REVIEW

**Comment by pierreneter** [comment]
Related to #6931

File: docs/pattern/form.en-US.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
+---
+order: 2
+title:
+  zh-CN: 表单
+  en-US: Form
+---
```
Comment (pierreneter) [inline docs/pattern/form.en-US.md]: Just keep title of this file's language. `title: Form`

File: docs/pattern/form.zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
+---
+order: 2
+title:
+  zh-CN: 表单
+  en-US: Form
+---
```
Comment (pierreneter) [inline docs/pattern/form.zh-CN.md]: And here

**Review by pierreneter — COMMENTED** [review]
Nice translation

File: docs/pattern/form.en-US.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+A form is an important human machine interface to obtain user information. It's necessary to consider what questions should be asked according to the expected answers.
```
Comment (zachguo) [inline docs/pattern/form.en-US.md]: "作为获取用户输入的重要交互方式，表单也承担将问题和答案进行配对的角色。"
The translation is a bit verbose. Taking first sentence as an example, we can just do 
'As an important interface to obtain user input, form plays a role of matching answers to questions.'

File: docs/pattern/form.en-US.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+In the design of a form, it's recommended to:
+
+1. Let a user knows exactly what information is required and why he needs to provide this information.
```
Comment (zachguo) [inline docs/pattern/form.en-US.md]: `a user` should be `an user`. I recommend using plural form `users` so you don't need to deal `he/she`.

Comment (zachguo) [inline docs/pattern/form.en-US.md]: I would rather make bullet points shorter, something like 
```
1. Make sure users know what is required to enter, and why
...
2. Provide rich context to help users complete the form
...
3. Error-sensitive and fault-tolerant
...
```

Comment (zachguo) [inline docs/pattern/form.en-US.md]: I think rest of the translation can also be improved this way, by making it less verbose. However, it's just my personal opinion anyway.

**Review by zachguo — CHANGES_REQUESTED** [review]

File: docs/pattern/form.en-US.md (hunk 4 above)
Comment (shenlin192) [inline docs/pattern/form.en-US.md]: Thank you for your professional suggestions!
I will correct them. By the way, `a user` is correct for me.

File: docs/pattern/form.zh-CN.md (hunk 2 above)
Comment (shenlin192) [inline docs/pattern/form.zh-CN.md]: Thank you pierreneter. I will correct them.

File: docs/pattern/form.en-US.md (hunk 4 above)
Comment (zachguo) [inline docs/pattern/form.en-US.md]: > I will correct them. By the way, `a user` is correct for me.
@shenlin192 You are right! 🤦‍♂️

File: docs/pattern/form.zh-CN.md [hunk 5]
```diff
@@ -2,7 +2,6 @@
 order: 2
 title:
   zh-CN: 表单
-  en-US: Form
```
Comment (afc163) [inline docs/pattern/form.zh-CN.md]:
```
title: 表单
```

File: docs/pattern/form.en-US.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
+---
+order: 2
+title:
+  en-US: Form
```
Comment (afc163) [inline docs/pattern/form.en-US.md]:
```
title: Form
```

**Comment by shenlin192** [comment]
I just changed the title of file `docs/pattern/form.zh-CN.md` and `docs/pattern/form.en-US.md` but I don't know why the continuous-integration fails. Is there something I did wrong? If yes, what should I do?

**Pull request merged**